 URL : "https://github.com/yingshaoxo/Telegram-with-QQ/issues/1" TITLE : 运行 qq.py 之后报错 BODY : root@vultr telegram-with-qq python3.6 qq.py
traceback most recent call last : file qq.py , line 105, in <module> from qqbot import qqbotslot as qqbs, runbot
modulenotfounderror: no module named 'qqbot'